src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -384,9 +384,13 @@ such that a permission bit being set in the result of <<GCPERM>>
 implies that all bits for permissions upon which it depends will also be set.
 
 For the base set of permissions just defined, the following rules apply.
-Extensions that define new permission bits (such as
-<<section_cheri_hybrid_ext,{cheri_default_ext_name}>> and <<section_zylevels1>>)
-may also introduce new dependency constraints.
+Extensions that define new permission bits may also introduce new dependency constraints.
+Currently defined examples are:
+
+. <<section_cheri_hybrid_ext,{cheri_default_ext_name}>>, see <<sec_m_bit_hybrid_rule>>
+. <<section_zylevels1>>, see <<zylevels1-clrperm-rules>>
+. <<section_zyseal>>, see <<zyseal_ap_field>>
```

Comment:
I think this xref is broken in the non-standalone build? might need an ifdef here?